test/helper.js

```diff
@@ -1,10 +1,12 @@
 var assert = require('assert');
-var debug = require('debug')('strong-pm:test');
 var path = require('path');
 var util = require('util');
 
 require('shelljs/global');
 
+// XXX should be a way to ignore specific shelljs globals
```

Comment:
But note that what I really want is to get this onto master ASAP, so linting always happens on test, and this doesn't happen by accident: https://github.com/strongloop/strong-pm/blame/3bafbe266c800bec7c1203570bcfcd081615fec4/test/test-service-manager.js#L12-L13

(fixed in this branch)